### ISSUE TITLE
[SRC] Official/latest Minix 3 source code

### ISSUE THREAD

**Issue by Kochise** [o-oconnell/minixfromscratch#6]
https://github.com/Stichting-MINIX-Research-Foundation/minix : check tags

**Comment by FabioLolix** [comment]
Indeed https://github.com/o-oconnell/minixfromscratch/commit/0f81584e1f08acee94b6a6e254c4bc510b0e5eaa#commitcomment-101827875

**Comment by Kochise** [comment]
Could've used it as a submodule, maybe...

**Comment by zjb1001** [comment]
**Environment:** WSl2 in Win10

**navigate to minix-3.1.0/tools to "make image"**, there is error as follows:

cd ../include && exec make - install
make[1]: Entering directory '/home/dp/GitPlayground/minixfromscratch/minix-3.1.0/include'
rm -rf /usr/include
mkdir -p /usr/include
cpdir . /usr/include
make[1]: cpdir: No such file or directory
make[1]: *** [Makefile:11: install] Error 127
make[1]: Leaving directory '/home/dp/GitPlayground/minixfromscratch/minix-3.1.0/include'
make: *** [Makefile:72: includes] Error 2

**Comment by o-oconnell** [comment]
Yeah I'll refactor this and put a reference to it maybe, it's been useful to have the code complete without mounting anything as a reference a few times though. You will not be able to compile this with gcc, you will need to recompile the image from within QEMU using the compiler that comes with the distribution, ACK

**Comment by Kochise** [comment]
> You will not be able to compile this with gcc, you will need to recompile the image from within QEMU using the compiler that comes with the distribution, ACK

Seriously ? ACK in 2023 ? This is a... HACK !
